universal-application-tool-0.0.1/conf/evolutions/default/3.sql

```diff
@@ -16,7 +16,8 @@ create table if not exists questions (
   description varchar,
   question_type varchar,
   question_text jsonb,
-  question_help_text jsonb
+  question_help_text jsonb,
+  question_options jsonb
```

Comment:
Suggestion (blocking): the safest way to add a column would be to add a new script that adds the new column to the table.

Reply:
Good to know! I think I did this, mind taking another look?

Reply:
Looks good!